### DISCUSSION TITLE
Error: ENOENT: no such file or directory

### DISCUSSION THREAD

**Discussion by iasadkhan1** [vercel/community#446] [Show & Tell]
![image](https://user-images.githubusercontent.com/92291001/155502460-6cc6b605-8679-4b45-8746-df1dd2f29f0f.png)

I deployed an api on vercel but whenever sending post request its giving me this error? does anyone know how to fix it??

**Comment by swarnava** [comment]
Hey, 

This thread might help: https://stackoverflow.com/questions/20753550/enoent-no-such-file-or-directory
